Financely/basic_app/stock_data.py

```diff
@@ -11,6 +11,7 @@ def candlestick_data(ticker):
     # newdata= data.to_dict()
     # print(newdata)'%Y-%m-%d'
     old  =  aapl.history(start="2001-05-21", end=now)
+    new = aapl.hisory(start="200020")
```

Comment:
**Issue**: Invalid date format

**Reason**: The date string '200020' is not a valid date format. It should be in the format 'YYYY-MM-DD'.

**Suggestion**: Change 'aapl.history(start="200020")' to 'aapl.history(start="2000-01-01")'